docs/source/cupy-reference/difference.rst

```diff
@@ -0,0 +1,55 @@
+Difference between CuPy and NumPy
+=================================
+
+The interface of CuPy is designed to obey the interface of NumPy.
+However it has some differenece.
+
+
+Cast behavior from float to integer
+-----------------------------------
+
+Some casting behavior from float to integer is not defined.
+For example from a negative float to unsigned integer and infinity to integer.
+It depends on your CPU.
```

Comment:
It seems to me that the difference of behaviors within NumPy is not relevant to the readers of this document because they are interested in the difference of NumPy and CuPy. Of course, this behavior of NumPy affects the difference between the two because it makes CuPy developers (like us) harder to mimic the behavior of NumPy. But it is not relevant to CuPy users.